<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrameMain</class>
 <widget class="QFrame" name="FrameMain">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>660</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>CodingDojo</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>326</width>
     <height>366</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>105</y>
      <width>256</width>
      <height>126</height>
     </rect>
    </property>
    <property name="value" stdset="0">
     <double>300.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="startTimerButton">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>315</y>
      <width>122</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetTimerButton">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>315</y>
      <width>122</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>35</y>
      <width>226</width>
      <height>46</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TIMER</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>505</x>
     <y>615</y>
     <width>122</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>335</x>
     <y>10</y>
     <width>286</width>
     <height>586</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="KLed" name="kledGreen">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>100</y>
      <width>136</width>
      <height>136</height>
     </rect>
    </property>
    <property name="state">
     <enum>KLed::Off</enum>
    </property>
   </widget>
   <widget class="KLed" name="kledYellow">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>260</y>
      <width>141</width>
      <height>136</height>
     </rect>
    </property>
    <property name="state">
     <enum>KLed::Off</enum>
    </property>
    <property name="color">
     <color>
      <red>255</red>
      <green>255</green>
      <blue>0</blue>
     </color>
    </property>
   </widget>
   <widget class="KLed" name="kledRed">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>420</y>
      <width>136</width>
      <height>136</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="state">
     <enum>KLed::Off</enum>
    </property>
    <property name="color">
     <color>
      <red>255</red>
      <green>0</green>
      <blue>0</blue>
     </color>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>226</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Running</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="configureButton">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>610</y>
     <width>122</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Configure</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLed</class>
   <extends>QWidget</extends>
   <header>kled.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>FrameMain</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>565</x>
     <y>629</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
